### PR TITLE
Adding backup and restore paragraph

### PR DESCRIPTION
**Pull request by dockerymick**
https://github.com/stolostron/backlog/issues/25148

### CODE REVIEW

**Review by oafischer — APPROVED** [review]
Looks good to me!

**Review by birsanv — APPROVED** [review]
lgtm

**Review by birsanv — CHANGES_REQUESTED** [review]
the new content looks good but it's still missing https://github.com/stolostron/cluster-backup-operator#extending-backup-data and https://github.com/stolostron/cluster-backup-operator#resources-restored-at-managed-clusters-activation-time

**Comment by dockerymick** [comment]
@birsanv thanks, I think i have added the correct info now. When you have a chance will you review again. @oafischer will you also rereview?

**Review by birsanv — APPROVED** [review]
lgtm thanks Mikela !

**Comment by dockerymick** [comment]
Thank you! @birsanv

File: backup_restore/backup_and_restore.adoc [hunk 1]
```diff
@@ -101,6 +104,84 @@ The operator defines the `backupSchedule.cluster.open-cluster-management.io` res
...
+[source,yaml]
+----
+apiVersion: my.group/v1alpha1
+kind: MyResource
+metadata:
+  labels:
+    cluster.open-cluster-management.io/backup: cluster-activation
+----
+
+Aside of these activation data resources that are identified by using the `cluster.open-cluster-management.io/backup: cluster-activation` label and stored by the `acm-resources-generic-schedule` backup, the cluster backup and restore operator includes a few resources in the activation set, by default. The following resources are backed up by the `acm-managed-clusters-schedule` backup:
```
Comment (oafischer) [inline backup_restore/backup_and_restore.adoc]: Aside from these... (?)

**Comment by oafischer** [comment]
Looks good @dockerymick, just left a small comment.

**Comment by dockerymick** [comment]
@oafischer @birsanv Thanks for yall review! Merging this PR

**Pull request merged**